graphql/resolvers.go

```diff
@@ -1064,6 +1067,49 @@ func (r *mutationResolver) SaveSubscription(ctx context.Context, subscription re
 	return true, nil
 }
 
+func (r *mutationResolver) UpdateUserSettings(ctx context.Context, userSettings *restModel.APIUserSettings) (bool, error) {
+	usr := route.MustHaveUser(ctx)
+	adminSettings, err := evergreen.GetConfig()
+	if err != nil {
+		return false, errors.Wrap(err, "Error retrieving Evergreen settings")
+	}
+	changedSettings, err := restModel.ApplyUserChanges(usr.Settings, *userSettings)
+	if err != nil {
+		return false, InternalServerError.Send(ctx, fmt.Sprintf("problem applying user settings: %s", err))
+	}
+	userSettingsInterface, err := changedSettings.ToService()
+	if err != nil {
+		return false, InternalServerError.Send(ctx, fmt.Sprintf("Error parsing user settings: %s", err))
+	}
+	UserSettings, ok := userSettingsInterface.(user.UserSettings)
```

Comment:
This variable is private so it should be lowercase. Also I would avoid naming it the same as the type